packages/create-block/lib/templates.js

```diff
@@ -24,62 +24,56 @@ const templates = {
 			license,
 			version,
 		},
-		outputFiles: {
-			'.editorconfig': 'editorconfig',
-			'editor.css': 'editor-css',
-			'index.js': 'es5/index-js',
-			'$slug.php': 'es5/plugin-php',
-			'style.css': 'style-css',
-		},
+		outputFiles: [ '.editorconfig', 'editor.css', 'index.js', '$slug.php', 'style.css' ],
```

Comment:
Still think we could just iterate over (recurse into) what exists in the folder itself, rather than explicitly list them (seems redundant). Basically seems like the idea is to copy the template folder, then apply the templating and rename the file to remove the `.mustache` suffix.

Could be done separately.

Reply:
I didn’t fully understood it before. I like the idea a lot and I’ll explore options. Really cool 👍